tensorboard/backend/application.py

```diff
@@ -238,6 +242,7 @@ def TensorBoardWSGIApp(
         multiplexer=deprecated_multiplexer,
         assets_zip_provider=assets_zip_provider,
         plugin_name_to_instance=plugin_name_to_instance,
+        sampling_hints=_parse_samples_per_plugin(flags),
```

Comment:
Parsing this twice felt easier than adding an extra parameter to this
method, but I’d be happy to change that if others object.